Error: Cannot find module 'caliper-core'

<!--- Provide a general summary of the issue in the Title above -->

## Context
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
error: [iroha.js]: Error: Cannot find module 'caliper-core'
info: [local-client.js]: Client[30782] encountered an error: Error: Failed when finding access point or user key
    at Iroha.getContext (/usr/lib/node_modules/caliper-cli/node_modules/caliper-iroha/lib/iroha.js:391:35)
    at Blockchain.getContext (/usr/lib/node_modules/caliper-cli/node_modules/caliper-core/lib/blockchain.js:74:33)
    at CaliperLocalClient.doTest (/usr/lib/node_modules/caliper-cli/node_modules/caliper-core/lib/client/caliper-local-client.js:252:49)
    at process.on (/usr/lib/node_modules/caliper-cli/node_modules/caliper-iroha/lib/irohaClientWorker.js:40:46)
    at emitTwo (events.js:126:13)
    at process.emit (events.js:214:7)
    at emit (internal/child_process.js:762:12)
    at _combinedTickCallback (internal/process/next_tick.js:142:11)
    at process._tickCallback (internal/process/next_tick.js:181:9)
error: [defaultTest]: ------ Failed 'open' testing with the following error ------
    Error: Client encountered error:Error: Failed when finding access point or user key
    at ChildProcess.<anonymous> (/usr/lib/node_modules/caliper-cli/node_modules/caliper-core/lib/client/client-util.js:102:36)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at emit (internal/child_process.js:762:12)
    at _combinedTickCallback (internal/process/next_tick.js:142:11)
    at process._tickCallback (internal/process/next_tick.js:181:9)
info: [defaultTest]: ####### Testing 'query' #######
info: [defaultTest]: ------ Test round 2 ------
info: [client-util.js]: Waiting for 0 clients to be ready... 
info: [client-util.js]: 0 clients ready, starting test phase
info: [local-client.js]: txUpdateTime: 1000
error: [iroha.js]: Error: Cannot find module 'caliper-core'
info: [local-client.js]: Client[30782] encountered an error: Error: Failed when finding access point or user key
    at Iroha.getContext (/usr/lib/node_modules/caliper-cli/node_modules/caliper-iroha/lib/iroha.js:391:35)
    at Blockchain.getContext (/usr/lib/node_modules/caliper-cli/node_modules/caliper-core/lib/blockchain.js:74:33)
    at CaliperLocalClient.doTest (/usr/lib/node_modules/caliper-cli/node_modules/caliper-core/lib/client/caliper-local-client.js:252:49)
    at process.on (/usr/lib/node_modules/caliper-cli/node_modules/caliper-iroha/lib/irohaClientWorker.js:40:46)
    at emitTwo (events.js:126:13)
    at process.emit (events.js:214:7)
    at emit (internal/child_process.js:762:12)
    at _combinedTickCallback (internal/process/next_tick.js:142:11)
    at process._tickCallback (internal/process/next_tick.js:181:9)
error: [defaultTest]: ------ Failed 'query' testing with the following error ------
    Error: Client encountered error:Error: Failed when finding access point or user key
    at ChildProcess.<anonymous> (/usr/lib/node_modules/caliper-cli/node_modules/caliper-core/lib/client/client-util.js:102:36)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at emit (internal/child_process.js:762:12)
    at _combinedTickCallback (internal/process/next_tick.js:142:11)
    at process._tickCallback (internal/process/next_tick.js:181:9)
info: [caliper-flow]: ---------- Finished Test ----------

error: [caliper-flow]: Error: TypeError: Cannot read property 'unshift' of undefined
    at Report.printResultsByRound (/usr/lib/node_modules/caliper-cli/node_modules/caliper-core/lib/report/report.js:133:32)
    at Object.module.exports.run (/usr/lib/node_modules/caliper-cli/node_modules/caliper-core/lib/caliper-flow.js:94:16)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
info: [demo.js]: [Transaction Info] - Submitted: 0 Succ: 0 Fail:0 Unfinished:0

## Expected Behavior
<!--- Tell us what should happen -->
when I use caliper test hyperledger iroha
caliper benchmark run --caliper-workspace ./packages/caliper-samples --caliper-benchconfig benchmark/simple/config-iroha.yaml --caliper-networkconfig network/iroha/simplenetwork/iroha.json

## Actual Behavior
<!--- Tell us what happens instead -->

## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug include code to reproduce, if relevant -->
1.
2.
3.
4.

## Existing issues
<!-- Have you searched for any existing issues or are their any similar issues that you've found? -->
- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-caliper)
- [ ] [GitHub Issues](https://github.com/hyperledger/caliper/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/caliper)

<!-- please include any links to issues here -->

## Context
<!--- How has this bug affected you? What were you trying to accomplish? -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used:
* Environment name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
* Link to your project:

--- a/packages/caliper-iroha/lib/iroha.js
+++ b/packages/caliper-iroha/lib/iroha.js
@@ -387,7 +387,7 @@ class Iroha extends BlockchainInterface {
             this.grpcQueryClient = new QueryService_v1Client(context.torii, grpc.credentials.createInsecure());
             return Promise.resolve(context);
         }catch(err) {
-            logger.error(err);
+            logger.error(`Error within getContext: ${err.stack ? err.stack : err}`);
             return Promise.reject(new Error('Failed when finding access point or user key'));
         }
     }

--- a/packages/caliper-samples/src/contract/iroha/simple/factory.js
+++ b/packages/caliper-samples/src/contract/iroha/simple/factory.js
@@ -15,9 +15,6 @@
 
 'use strict';
 
-const CaliperUtils = require('caliper-core').CaliperUtils;
-const logger = CaliperUtils.getLogger('factory.js');
-
 // TODO: two or more commands
 const open = function(context, args) {
 
@@ -49,7 +46,7 @@ const simple = function(context, args) {
         }
     }
     catch(err){
-        logger.error(err);
+        console.error(err);
         return {};
     }
 };

